src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -470,13 +482,18 @@ type Comment {
   replyCount: Int
 
   """
-  replies will return the replies to this comment.
+  replies will return the replies to this Comment.
   """
   replies(
     first: Int = 10
     orderBy: COMMENT_SORT = CREATED_AT_DESC
     after: Cursor
   ): CommentsConnection
+
+  """
+  editing returns details about the edit status of a Comment.
+  """
+  editing: EditInfo
```

Comment:
What do you think of adding it directly to `Comment` instead of using a separate type `EditInfo`? Makes it slightly easier to consume on the client. Just a thought.

Reply:
Besides that, shouldn't this be `EditInfo!`?

Reply:
I can change it to `EditInfo!`, but without repeating the same logic twice, shouldn't it be the same easiness level to access `comment.editInfo.?` than accessing `comment.?`?

Reply:
I guess it's fine. Optimistic updates are a bit more code first getting the comment, then getting the linked record and then setting the value. The more nested it is, the more cumbersome it is with the vanilla relay api.